web/src/lib/types/types.ts

```diff
@@ -18,57 +18,63 @@ export type OperateDisplayContent = {
   entityIdErrorNotRegisteredMd: string;
 };
 
-type TextFieldContent = {
+export type TextFieldContent = {
   contentMd: string;
-  placeholder: string;
+  placeholder?: string;
 };
+export type RadioFieldContent = { contentMd: string; radioButtonYesText: string; radioButtonNoText: string };
 
-export type ProfileDisplayContent = {
-  hasExistingBusiness: { contentMd: string; radioButtonYesText: string; radioButtonNoText: string };
+export type LegalFieldContent = {
+  contentMd: string;
+  optionContent: Record<string, string>;
+};
+export type FlowType = "OWNING" | "STARTING";
+
+export type UserContentType = FlowType | "PROFILE";
+export interface IndustryFieldContent extends TextFieldContent {
+  specificHomeContractorMd: string;
+  specificEmploymentAgencyMd: string;
+  specificLiquorQuestion: RadioFieldContent;
+  specificHomeBasedBusinessQuestion: RadioFieldContent;
+}
+
+export type StartingFlowContent = {
+  hasExistingBusiness: RadioFieldContent;
   businessName: TextFieldContent;
-  industry: {
-    contentMd: string;
-    placeholder: string;
-    specificHomeContractorMd: string;
-    specificEmploymentAgencyMd: string;
-    specificLiquorQuestion: {
-      contentMd: string;
-      radioButtonYesText: string;
-      radioButtonNoText: string;
-    };
-    specificHomeBasedBusinessQuestion: {
-      contentMd: string;
-      radioButtonYesText: string;
-      radioButtonNoText: string;
-    };
-  };
-  legalStructure: {
-    contentMd: string;
-    optionContent: Record<string, string>;
-  };
-  municipality: {
-    contentMd: string;
-    placeholder: string;
-  };
-  notes: {
-    contentMd: string;
-  };
-  taxId: {
-    contentMd: string;
-  };
-  employerId: {
-    contentMd: string;
-  };
-  entityId: {
-    contentMd: string;
-  };
-  certifications: { contentMd: string; placeholder: string };
-  existingEmployees: {
-    contentMd: string;
-    placeholder: string;
-  };
+  industry: IndustryFieldContent;
+  municipality: TextFieldContent;
+  legalStructure: LegalFieldContent;
 };
 
+export type ProfileContent = {
+  businessName: TextFieldContent;
+  industry: IndustryFieldContent;
+  municipality: TextFieldContent;
+  legalStructure: LegalFieldContent;
+  notes: TextFieldContent;
+  taxId: TextFieldContent;
+  entityId: TextFieldContent;
+  employerId: TextFieldContent;
+};
+
+export type OwningFlowContent = {
+  hasExistingBusiness: RadioFieldContent;
+  businessName: TextFieldContent;
+  industry: IndustryFieldContent;
+  entityId: TextFieldContent;
+  certifications: TextFieldContent;
+  municipality: TextFieldContent;
+  existingEmployees: TextFieldContent;
+};
+export interface UserDisplayContent extends StartingFlowContent, OwningFlowContent, ProfileContent {}
```

Comment:
the umbrella term "UserDisplayContent" isn't exactly what I'd expect... trying to think of an alternative suggestion... but since we already have a User object that's unrelated, it feels weird